app/assets/stylesheets/views/_homepage.scss

```
@@ -257,20 +237,32 @@
   }
 }
 
-// Created a custom section and header pattern to
-// - display a border-top of 2px (to replace .home-divider)
-// - margin-bottom of 30px (both mobile and desktop)
-// - padding-top to create a gap between border and heading
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+}
 
 .homepage-section {
-  margin: govuk-spacing(4) 0 govuk-spacing(8);
+  padding: govuk-spacing(4) 0 govuk-spacing(6);
+
+  &.homepage-section.homepage-section--links-and-search {
```

Is the extra class necessary?
```suggestion
  &.homepage-section--links-and-search {
```

You could also place these modifiers below `homepage-section` as separate class declarations.

---

Good spot! That can be removed.